[2025-01-21T14:50:18.430+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 04_task_sequence_morning.sock_left manual__2025-01-21T14:50:06.826875+00:00 [queued]>
[2025-01-21T14:50:18.460+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 04_task_sequence_morning.sock_left manual__2025-01-21T14:50:06.826875+00:00 [queued]>
[2025-01-21T14:50:18.460+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-21T14:50:18.497+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): sock_left> on 2025-01-21 14:50:06.826875+00:00
[2025-01-21T14:50:18.514+0000] {standard_task_runner.py:60} INFO - Started process 704 to run task
[2025-01-21T14:50:18.525+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '04_task_sequence_morning', 'sock_left', 'manual__2025-01-21T14:50:06.826875+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/04_task_sequence_dag.py', '--cfg-path', '/tmp/tmp0i5cxjs2']
[2025-01-21T14:50:18.529+0000] {standard_task_runner.py:88} INFO - Job 6: Subtask sock_left
[2025-01-21T14:50:18.563+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-21T14:50:18.689+0000] {task_command.py:423} INFO - Running <TaskInstance: 04_task_sequence_morning.sock_left manual__2025-01-21T14:50:06.826875+00:00 [running]> on host d3a2c9f62df6
[2025-01-21T14:50:18.836+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='04_task_sequence_morning' AIRFLOW_CTX_TASK_ID='sock_left' AIRFLOW_CTX_EXECUTION_DATE='2025-01-21T14:50:06.826875+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-21T14:50:06.826875+00:00'
[2025-01-21T14:50:18.838+0000] {logging_mixin.py:188} INFO - Pull on left sock
[2025-01-21T14:50:18.839+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-21T14:50:18.856+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=04_task_sequence_morning, task_id=sock_left, execution_date=20250121T145006, start_date=20250121T145018, end_date=20250121T145018
[2025-01-21T14:50:18.902+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-21T14:50:18.954+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
